pkgs/os-specific/linux/pam_rssh/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  pam,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "pam_rssh";
+  version = "unstable-2023-01-12";
+
+  src = fetchFromGitHub {
+    owner = "z4yx";
+    repo = pname;
+    rev = "773823b9a8605436e5bcfabd1c3ff8deb8503f43";
+    hash = "sha512-rAr4ugo+IaxOKXtfEq9stK4edodork2uz10trHDv9MN4Oc296uDQjLh8Sle04Z4mbW56rWlT8eHrW6YpqGHFpw==";
```

Comment:
Not sure why you are using sha512 here

Reply:
Why not?

Reply:
I am not sure if this is supported by the update bot by default.

Reply:
It is supported, but sha512 will be downgraded to sha256. See [this update log][1] for an example.

I opened an issue on nixpkgs-update to track this: https://github.com/ryantm/nixpkgs-update/issues/350

[1]: https://r.ryantm.com/log/wasabibackend/2023-04-08.log